appservice/multiplexer.py

```diff
@@ -17,6 +17,7 @@
 SESSION_INSTANCE_DOMAIN = os.getenv('SESSION_INSTANCE_DOMAIN', '')
```

Comment:
Small typo in the commit message `hte` => `the`

Reply:
fixed